### ISSUE TITLE
Add ignore: ["consecutive-duplicates-with-different-values"] option to declaration-block-no-duplicate-properties

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#1759]
Ref: https://github.com/CSSLint/csslint/wiki/Disallow-duplicate-properties

Proposal to add an option similar to [`"consecutive-duplicates"`](https://github.com/stylelint/stylelint/blob/master/src/rules/declaration-block-no-duplicate-properties/README.md#ignore-consecutive-duplicates), but less permissive.

**Comment by evilebottnawi** [comment]
sgtm, vote for this option

**Comment by sendilkumarn** [comment]
👍 it is essential now

**Comment by ntwb** [comment]
SGTM

**Comment by delorge** [comment]
I'd like to implement this option. Please assign this to me.

**Comment by evilebottnawi** [comment]
@delorge let's go :+1:

**Comment by jeddy3** [comment]
@delorge  Good luck and thanks for helping out. You'll find plenty of information in the [developer guide](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md) to help you get started.

**Comment by delorge** [comment]
Before I commit any further changes, I want to make things clear: is it necessarily for 'consecutive-duplicates-with-different-values' to check if duplicates are consecutive as it get tested in "consecutive-duplicates"? I assume that these options are independent and any of them should warn user about incorrect duplicates order. 

Having this behavior given, I've provided appropriate tests, but as @davidtheclark reasonably mentioned, they seem to repeat themselves.

Finally, I thought maybe we should deprecate "consecutive-duplicates" in favor of "consecutive-duplicates-with-different-values"? IMO "consecutive-duplicates-with-different-values" makes more sense (we basically don't need to ignore properties with identical values)

P.S. I haven't seen any checks for multiple duplicates. Shall we implement that as well?
```css
.example-selector {
   font-size: 16px;
   font-size: 16px;
   font-size: 1rem;
}
```

**Comment by evilebottnawi** [comment]
@davidtheclark close?

**Comment by jeddy3** [comment]
Done in https://github.com/stylelint/stylelint/issues/1775
